app/models/like.rb

```diff
@@ -1,2 +1,11 @@
 class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :comment
```

Comment:
- [ Required ] As suggested by the previous reviewer it would be nice if you specify in this model that the `Like` belongs to a `user` and a `post`. Please see the example code below for a suggestion on how to achieve this.

## Example code
```ruby
class Like < ApplicationRecord
  belongs_to :post
  belongs_to :user
end
```